Gopkg.lock

```diff
@@ -1160,10 +1152,10 @@
     "github.com/pkg/errors",
     "github.com/sirupsen/logrus",
     "github.com/spf13/cobra",
+    "github.com/spf13/pflag",
```

Comment:
where is this dependency coming from?

Reply:
this is just an artifact added by the dep solver. the actual pflag dependency didn't change here